nimbus/src/nimbus.idl

```diff
@@ -1,6 +1,8 @@
 namespace nimbus {};
 dictionary AppContext {
+    string app_name;
     string app_id;
+    string channel;
```

Comment:
Yeah, I'm not sure if the IDL is the best place for that or not. On one hand, this is really the key in defining the exposed API, but on the other I don't know that I would go looking in the IDL for this sort of information. This seems like it would be better documented in `matcher.rs` along with the `AppContext` struct definition.  What do you think?

Reply:
I suspect the most important places to have it defined are a) in the code so maintainers handle it correctly and b) in the exposed API so that clients handle it correctly.  Maybe start by putting it in rustdoc comments in matcher.rs, on the assumption that at some point we'll start generating docs from that?

Reply:
That's where I was expecting to add the docs also.  Commit added with documentation added for the `AppContext` object.